### DISCUSSION TITLE
モンスター追加調整案

### DISCUSSION THREAD

**Discussion by deskull-m** [sikabane-works/bakabakaband#987] [Ideas]
変愚じゃアレ過ぎたり冒険的過ぎるものもある程度カバー。募集中。

**Comment by Hourier** [comment]
素案しかないですが投げておきます
3体に分裂します (変愚では未実装)

N:XXX0:三国志の『義兄弟』
F:CAN_SPEAK
「我ら三人、生まれし日、時は違えども、兄弟の契りを結びし時からは心を同じくして助け合い、困窮する者達を救わん！」
「同年同月同日に生まれずとも、同年同月同日に死せん事を願わん！」

もちろん分離先は劉備、関羽、張飛

**Comment by Hourier** [comment]
ちょっとアレな方
『ライカ』を実装したのでついでに
これも素案だけです
タチャンカ('q')とか召喚してもいいかも

N:YYY0:革命的『元帥』
E:Marshal Sovietskogo Soyuza, the Revolutionary General
S:1\_IN\_4 | BR_COLD
D$:They are loyal subordinates of Uncle Iosif.
D:ヨシフおじさんの忠実な部下だ。
分離先：YYY1
分離先：YYY2

N:YYY1:赤き同志『ヴォロシーロフ』
E:Voroshilov, the Red General
F:UNIQUE | FORCE_MAXHP | MALE
S:1\_IN\_4 | BR_COLD

N:YYY2:赤き同志『イェゴロフ』
E:Yegorov, the Red General
F:UNIQUE | FORCE_MAXHP | MALE
S:1\_IN\_4 | BR_COLD

**Comment by Hourier** [comment]
引き続きアレな方
最後はその筋の人には有名ですね

N:ZZZ0:反革命的『元帥』
D:ルビャンカの地下はシベリアだけでなく鉄獄にも繋がっているらしい。
F:UNIQUE | FORCE_MAXHP | MALE | UNDEAD
S:1\_IN\_4 | BR_COLD
分離先：ZZZ1
分離先：ZZZ2
分離先：ZZZ3

N:ZZZ1:反革命的元帥『ブリュヘル』
E:Blyukher, the Anti-Revolutionary General
F:UNIQUE | FORCE_MAXHP | MALE | UNDEAD
S:1\_IN\_4 | BR_COLD
D:まだ彼が生きていた頃、そのような汚名を着せられた。

N:ZZZ2:無能元帥『ブジョーンヌイ』
E:Budyonny, the Incompetence General
F:UNIQUE | FORCE_MAXHP | MALE | UNDEAD
S:1_IN_4 | BR_COLD
D:まだ彼が生きていた頃、そのような汚名を着せられた。

N:ZZZ3:帝国のスパイ『トゥハチェフスキー』
E:Tukhachevsky, the Spy of Nazi
F:UNIQUE | FORCE_MAXHP | MALE | UNDEAD
S:1\_IN\_4 | BR_COLD
D:まだ彼が生きていた頃、そのような汚名を着せられた。

**Reply by deskull-m** [reply]
うーむ、どれも追々って所ですな。まあ備忘位にここにとどめておきます。

**Comment by Taro0910** [comment]
ゴッドマン、アーマージャックに続くロクでもないヒーロー。原作どおり荒野でも登場するキャラクターのつもりで。（五右衛門とかと同じ）
他にも出来れば
•特別な行動として既存の大鷲ユニークと同じ放り投げ
•矢を放つ際に「レッドアロー！！」
前述の放り投げで「レッドフォール！！」
ファイアボルトを放つ際「レッドナイーフ！！」
と表示する（マリオはスターを取った！　と同じタイプのメッセージ）
•monspeakで「レッドファイッ！！」「イヤッ！！」と喋らせる
以上３つの要素を入れて欲しいです。

[U] 赤いあいつ『レッドマン』/Redman (Red 'p') 
階層:24  レア度:7  加速:0  HP:800  AC:80  Exp:400
遥か銀河の赤い星から光に乗ってやって来た、平和を愛する心優しい男だ。
彼は通常地下 24 階で出現し、普通の速さで動いている。
この善良なる人間を倒すことは 1 レベルのキャラクタにとって 約3200.00 ポイントの経験となる。
彼は特別な行動をすることがある。
彼は光っている。
彼は炎に包まれている。
彼は威力 6d3 の射撃をすることがある。
彼は魔法を使うことができ、ファイア・ボルト(12+9d8)、サンダー・ボルト(10+4d8)、加速の呪文を唱えることがある(確率:1/4)。
それは AC80 の防御力と800 の体力がある。
彼はドアを打ち破り、水を渡り、弱いモンスターを倒すことができる。
彼は炎の耐性を持っている。
彼は恐怖を感じないし、混乱しないし、眠らされない。
彼は侵入者をかなり警戒しており、 400 フィート先から侵入者に気付くことがある。
彼は 4d6 のダメージで斬って燃やし、 4d6 のダメージで蹴って強力に攻撃し、4d6 のダメージで殴って強力に攻撃する。

**Reply by deskull-m** [reply]
https://github.com/deskull-m/bakabakaband/pull/6533
すんません、思い出したように気づきましたが、色々盛って実装してしまいましたわ……
